package.json

```diff
@@ -1,4 +1,5 @@
 {
- "name": "@components-studio/divriots.studio-story-renderer",
- "version": "0.0.0"
+  "name": "@components-studio/divriots.studio-story-renderer",
```

Comment:
I think it should be @divriots/studio-story-rendere

Reply:
@muryoh  :D

Reply:
Heh we'll fix it :+1: 
It's ok in the released package though

![image](https://user-images.githubusercontent.com/1789748/135056701-b8858777-d041-4c08-9379-f1c126332efc.png)

Reply:
ok 👍